app-modules/student-record-manager/src/Filament/Imports/StudentManageableImporter.php

```diff
@@ -0,0 +1,151 @@
+
+    public static function getColumns(): array
+    {
+        return [
+            ImportColumn::make('sisid')
+                ->label('Student ID')
+                ->requiredMapping()
+                ->example('########')
+                ->numeric(),
+            ImportColumn::make('otherid')
+                ->label('Other ID')
+                ->example('##########')
+                ->numeric(),
+            ImportColumn::make('first')
+                ->example('Jonathan'),
+            ImportColumn::make('last')
+                ->example('Smith'),
+            ImportColumn::make('full_name')
+                ->example('Jonathan Smith'),
+            ImportColumn::make('preferred')
+                ->example('John'),
+            ImportColumn::make('birthdate')
+                ->example('2024-10-21'),
+            ImportColumn::make('hsgrad')
+                ->rules(['min:1920', 'max:now()->addYears(25)->year'])
+                ->example('1920'),
+            ImportColumn::make('email')
+                ->rules(['required', 'email'])
+                ->example('johnsmith@gmail.com'),
+            ImportColumn::make('email_2')
+                ->rules(['email'])
+                ->example('johnsmith@hotmail.com'),
+            ImportColumn::make('mobile')
+                ->example('+1 (555) 555-5555'),
+            ImportColumn::make('phone')
+                ->example('+1 (555) 555-5555'),
+            ImportColumn::make('address')
+                ->example('123 Main St.'),
+            ImportColumn::make('address2')
+                ->example('Apt. 1'),
+            ImportColumn::make('address3')
+                ->example('xyz'),
+            ImportColumn::make('city')
+                ->example('Los Angeles'),
+            ImportColumn::make('state')
+                ->example('california'),
+            ImportColumn::make('postal')
+                ->example('83412'),
+            ImportColumn::make('sms_opt_out')
+                ->label('SMS opt out')
+                ->boolean()
+                ->rules(['boolean'])
+                ->example('no'),
+            ImportColumn::make('email_bounce')
+                ->boolean()
+                ->rules(['boolean'])
+                ->example('yes'),
+            ImportColumn::make('dual')
+                ->boolean()
+                ->rules(['boolean'])
+                ->example('yes'),
+            ImportColumn::make('ferpa')
+                ->label('FERPA')
+                ->boolean()
+                ->rules(['boolean'])
+                ->example('yes'),
+            ImportColumn::make('dfw')
+                ->label('DFW')
+                ->example('2024-10-21'),
+            ImportColumn::make('sap')
+                ->label('SAP')
+                ->boolean()
+                ->rules(['boolean'])
+                ->example('yes'),
+            ImportColumn::make('holds')
+                ->rules(['regex: [A-Z]{5}'])
```

Comment:
Please remove this rule. We cannot control or expect what the hold string will be like at all colleges.